87-percentiles.md

```diff
@@ -0,0 +1,46 @@
+# How do you calculate percentiles with Python/ NumPy?
+
+The nth percentile of a dataset is the value that cuts off the first n percent of the data values when all of the values are sorted from least to greatest.
+
+We can calculate percentiles with the following code:
+
+```python
+import numpy as np
+array = np.array([1, 2, 3, 4, 5])
+# return 50th percentile, e.g median
+percentile = np.percentile(array, 50) 
+print(percentile)
```

Comment:
Do you need this `import numpy as np` ?

Reply:
Not working: 
![image](https://user-images.githubusercontent.com/12773753/126868849-fbd85d8a-cebd-4e11-a41f-d99f09885776.png)